### ISSUE TITLE
tasks in block statements are not converted

### ISSUE THREAD

**Issue by zerwes** [zerwes/ansible-fqcn-converter#5]
block statements fail ...
example file : https://github.com/ONLYOFFICE/ansible-role-documentserver/pull/41/files#diff-3b500db77c1efb294f1b83db7f4c5174166d35fbfc6800441497b89cfbbdba5b

**Comment by zerwes** [comment]
commit 231cd42 fails
```diff
parsing file /home/zerwes/git/ansible-fqcn-converter/test-in-block.yml ..*..*
--- a//home/zerwes/git/ansible-fqcn-converter/test-in-block.yml
+++ b//home/zerwes/git/ansible-fqcn-converter/test-in-block.yml
@@ -1,6 +1,7 @@
 ---
 - name: test
-  copy:
+  ansible.builtin.copy:
     content: aaa
     dest: xyz
-    group: lorem
+# possible ambiguous replacement: group : awx.awx.group | ansible.builtin.group
+    awx.awx.group: lorem
```

**Comment by ryaner** [comment]
Another example of the same problem, this time with a bit not converted.
```
# FQCN Test Playbook
---
- name: FQCN test playbook
  hosts: localhost
  ansible.builtin.gather_facts: false
  tasks:
    - name: Sanity check required inputs
      assert:
        that:
          - dc is defined
        fail_msg:
          - "dc fail"
```
The `assert` isn't converted even though it is in the map.

**Comment by zerwes** [comment]
yes, this has the same cause as the block issue (or #3)
this is something I am working on ... (but due to "lights out doe to work overload" it might take some time ...)

**Comment by zerwes** [comment]
should be fixed with release 0.1.0
